src/utils.rb

```diff
@@ -0,0 +1,15 @@
+#: Common utilities for AoC solutions project
+
+module AocUtils
+
+  def AocUtils.file_to_array(filename)
+    res = []
+    File.open(filename) do |f|
+      f.each_line do |line|
```

Comment:
You can do `f.each_line.map { |l| l.strip.to_i }`

Reply:
Yep, much more elegant. Done in 6b32ef8.